src/neynar-api/neynar-api-client.ts

```diff
@@ -2249,6 +2250,66 @@ export class NeynarAPIClient {
     return await this.clients.v2.fetchRelevantFollowers(targetFid, viewerFid);
   }
 
+  /** 
+  * Returns a list of followers for a specific FID.
+  * @summary Retrieve followers for a given user
+  * @param {number} fid User who's profile you are looking at
+  * @param {Object} [options] - Optional parameters for customizing the response.
+  * @param {number} [options.viewerFid] Viewer who's looking at the profile.
+  * @param {FollowSortType} [options.sortType] Sort type for retrieving followers. Default is FollowSortType.DescChron
+  * @param {number} [options.limit] Number of results to retrieve (default 20, max 100)
+  * @param {string} [options.cursor] Pagination cursor.
+  *  omit this parameter for the initial request.
+  * 
+  * @returns {Promise<UsersResponse>} A promise that resolves to a `UsersResponse` object,
+  * 
+  * @example
+  * // Example: Retrieve followers for a user
+  * import { FollowSortType } from "@neynar/nodejs-sdk";
+  * 
+  * client.fetchUserFollowersV2(3,{limit: 5,viewerFid: 3}).then(response => {
+  *  console.log('User Followers:', response);
+  * });
+  * 
+  * For more information, refer to the [Neynar documentation](https://docs.neynar.com/reference/followers-v2).
+*/
+  public async fetchUserFollowersV2(
+    fid: number,
+    options?: { viewerFid?: number; sortType: FollowSortType; cursor?: string; limit?: number }
+  ): Promise<UsersResponse> {
+    return await this.clients.v2.fetchUserFollowersV2(fid, options);
+  }
+
+  /** 
+  * Returns a list of follows for a specific FID.
+  * @summary Retrieve follows for a given user
+  * @param {number} fid User who's profile you are looking at
+  * @param {Object} [options] - Optional parameters for customizing the response.
+  * @param {number} [options.viewerFid] Viewer who's looking at the profile.
+  * @param {FollowSortType} [options.sortType] Sort type for retrieving follows. Default is FollowSortType.DescChron
+  * @param {number} [options.limit] Number of results to retrieve (default 25, max 100)
+  * @param {string} [options.cursor] Pagination cursor.
+  *  omit this parameter for the initial request.
+  * 
+  * @returns {Promise<UsersResponse>} A promise that resolves to a `UsersResponse` object,
+  * 
+  * @example
+  * // Example: Retrieve follows for a user
+  * import { FollowSortType } from "@neynar/nodejs-sdk";
+  * 
+  * client.fetchUserFollowingV2(3,{limit: 5,viewerFid: 3}).then(response => {
+  * console.log('User Follows:', response);
+  * });
+  * 
+  * For more information, refer to the [Neynar documentation](https://docs.neynar.com/reference/following-v2).
+*/
+  public async fetchUserFollowingV2(
```

Comment:
@manan19 Should we name it like this given that v1 is called `fetchUserFollowing`, if both methods returning the same data converted to v2 objects, then I think the name make sense

cc @MrKevinOConnell to confirm v1 and v2 responses

Reply:
i believe fetchUserFollowing is returning v1 objects whereas fetchUserFollowingV2 is returning v2 objects @Shreyaschorge

Reply:
@MrKevinOConnell That's true, what I meant was apart from v1 and v2 objects is there anything else that has changed?

Reply:
no they are the same @Shreyaschorge 👍🏻